tests/unit/test_log_proxy_consumer.py

```diff
@@ -103,6 +99,41 @@ def test__initial_config_sections(self):
         expected_sections = {"positions", "scrape_configs", "server"}
         self.assertEqual(set(self.harness.charm._log_proxy._initial_config), expected_sections)
 
+    def test__initial_config_jobs(self):
+        expected_jobs = {"system", "syslog"}
+        self.assertEqual(
+            set(
+                map(
+                    lambda x: x["job_name"],
+                    self.harness.charm._log_proxy._initial_config["scrape_configs"],
+                )
+            ),
+            expected_jobs,
+        )
+
+    def test__initial_config_labels(self):
```

Comment:
Function name should have a behavioral phrasing, e.g.
- test all scrape jobs have all topology labels
- test all job names are either system or syslog

Reply:
This is to be consistent with existing test names.

Reply:
Currently this test is quite implicit. Consider breaking it down into
- test all scrape jobs have all topology labels
- test all job names are either system or syslog
- more? not sure what this test tests

Reply:
`test_base_config_includes_system_logs` seems to cover it